### PR TITLE
Create Android.mk

### PR DESCRIPTION
**Pull request by be-neth**
Use pull request #26 from @daktak and Fix Android.mk.

Tested with a ROM build. (LineageOS 7.1 for oneplus3)

### CODE REVIEW

**Comment by mar-v-in** [comment]
There is a typo two times `IchnaeaLnp` vs `IchnaeaNlp`

File: Android.mk [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := IchnaeaNlp
+LOCAL_MODULE_TAGS := optional
+LOCAL_PACKAGE_NAME := IchnaeaNlp
+
+ichnaea_root  := $(LOCAL_PATH)
+ichnaea_out   := $(OUT_DIR)/target/common/obj/APPS/$(LOCAL_MODULE)_intermediates
+ichnaea_build := $(ichnaea_root)/build
+ichnaea_apk   := build/outputs/apk/IchnaeaNlp-release-unsigned.apk
```
Comment (mar-v-in) [inline Android.mk]: Shouldn't this be `build/outputs/apk/IchnaeaNlpBackend-release-unsigned.apk`

Comment (be-neth) [inline Android.mk]: I have to try, but for me the apk name is linked to this variable:
LOCAL_PACKAGE_NAME := IchnaeaNlp

Comment (mar-v-in) [inline Android.mk]: The final apk name, as it ends up in `/system/app` indeed is defined by that variable, but the output of Gradle is not set by that variable but by the name of the directory. As you usually check out a repo with name IchnaeaNlpBackend in a directory called IchnaeaNlpBackend, it should end up with this name.

Comment (be-neth) [inline Android.mk]: Is it possible to change the gradle output name ?
(When building and integrating a ROM, you cannot deduce the name of the folder without looking at specific project Android.mk...)

Comment (mar-v-in) [inline Android.mk]: The solution would be to use a subfolder, as it is done with GmsCore.

**Pull request merged**